_plugins/spryker_documentation/Gemfile

```diff
@@ -0,0 +1,8 @@
+# frozen_string_literal: true
```

Comment:
Why do you need a custom gem?

Reply:
There is no other way for custom command plugin to be integrated only as a new gem. CLI not added if it just placed inside `_plugins` as `.rb` script.